### PR TITLE
Add test page for multiple miradors

### PR DESCRIPTION
**Pull request by aeschylus**
Part of #1782

### CODE REVIEW

File: src/lib/MiradorViewer.js [hunk 1]
```diff
@@ -8,24 +8,24 @@ import createStore from '../state/createStore';
...
 
-const store = createStore();
 /**
  * Default Mirador instantiation
  */
 class MiradorViewer {
   /**
    */
   constructor(config) {
+    this.store = createStore();
```
[the remark this answers is not in the document]
Comment (aeschylus) [reply src/lib/MiradorViewer.js]: I've ticketed this up here: https://github.com/ProjectMirador/mirador/issues/1844

I think for now we let this PR hang out as a testing ground for whatever we decide in a targeted PR for that issue.

[the remark this answers is not in the document]
Comment (cbeer) [reply src/lib/MiradorViewer.js]: I'm strongly 👎 to letting the PR hang out.  If it's a reasonable, incremental improvement, I'd rather get it merged and address #1844 when it bubbles up as a priority. If we think it's not a direction we'd consider, I'd rather see us close it entirely.

[the remark this answers is not in the document]
Comment (mejackreed) [reply src/lib/MiradorViewer.js]: Without active work on #1844, can we push forward here with this PR and use #1844 to better define this?

**Comment by mejackreed** [comment]
Shipping this so we can demo and move forward #1844

**Pull request merged**